### ISSUE TITLE
Suppress laser desktop cursor

### ISSUE THREAD

**Issue by shiftyscales** [Frooxius/NeosPublic#1498]
An issue I've encountered while using the Leap Motion, or leaving my controllers facing towards the desktop viewer is that the lasers are always-on, and steal focus of the cursor, preventing me from being able to use my cursor unless I disable my Leap/ensure the controllers are facing away from the desktop view.

Perhaps requirig the laser to click on the screen to take focus of the cursor could be beneficial in these scenarios?

**Comment by Frooxius** [comment]
The click might work, but it will also make it impossible to just hover over something without clicking something first. Does the laser move at all in this scenario? I could check if they're stationary and simply not move the cursor to the same position. But if there's even small movement that wouldn't work.

**Comment by shiftyscales** [comment]
Yes, because my hands are moving, the laser is sweeping across the screen while being tracked by hand tracking, and the controllers can wobble slightly while in my lap otherwise. @Frooxius

**Comment by Frooxius** [comment]
Where are the lasers coming from? Are they coming from your hands tracked with LeapMotion?

**Comment by shiftyscales** [comment]
Yes.
